app/src/main/java/com/space/quizapp/presentation/ui/points/QuizPointsFragment.kt

```diff
@@ -35,11 +32,12 @@ class QuizPointsFragment : QuizBaseFragment<QuizPointsViewModel>() {
     }
 
     private fun showLogOutDialog() {
-        showDialog(
-            R.layout.dialog_listener,
-            onPositiveButtonClick = {
-                navigateSafe(QuizPointsFragmentDirections.actionQuizPointsFragmentToQuizLogInFragment())
+        showDialog(R.layout.dialog_listener, onPositiveButtonClick = {
+            lifecycleScope {
+                viewModel.clearUserSession {
+                    navigateSafe(QuizPointsFragmentDirections.actionQuizPointsFragmentToQuizLogInFragment())
+                }
             }
```

Comment:
its not a good idea, try what I wrote before

Reply:
გასწორებულია https://github.com/iTarley/QuizApp-SpaceIntern/pull/5